### PR TITLE
Make NFT descriptor public

### PR DESCRIPTION
**Pull request by andreivladbrg**
Closes https://github.com/sablier-labs/v2-core/issues/763

### CODE REVIEW

File: src/abstracts/SablierV2Lockup.sol [hunk 1]
```diff
@@ -29,12 +29,8 @@ abstract contract SablierV2Lockup is
     /// @inheritdoc ISablierV2Lockup
     uint256 public override nextStreamId;
 
-    /*//////////////////////////////////////////////////////////////////////////
-                                  INTERNAL STORAGE
-    //////////////////////////////////////////////////////////////////////////*/
-
     /// @dev Contract that generates the non-fungible token URI.
```
Comment (smol-ninja) [inline src/abstracts/SablierV2Lockup.sol:32]: It should be 
```soldiity
/// @inheritdoc ISablierV2Lockup
```

File: src/interfaces/ISablierV2Lockup.sol [hunk 2]
```diff
@@ -134,6 +134,9 @@ interface ISablierV2Lockup is
     /// @notice Counter for stream ids, used in the create functions.
     function nextStreamId() external view returns (uint256);
 
+    /// @notice Contract that generates the non-fungible token URI.
```
Comment (smol-ninja) [inline src/interfaces/ISablierV2Lockup.sol:137]: How about this?

```solidity
/// @notice Retrieves the address of NFT descriptor contract that generates the non-fungible token URI.
```

**Review by smol-ninja — CHANGES_REQUESTED** [review]

File: src/abstracts/SablierV2Lockup.sol (hunk 1 above)
Comment (andreivladbrg) [reply src/abstracts/SablierV2Lockup.sol:32]: correct

File: src/interfaces/ISablierV2Lockup.sol (hunk 2 above)
Comment (andreivladbrg) [reply src/interfaces/ISablierV2Lockup.sol:137]: it is better like it is now

we usually use "Retrieves" word for getters that implement the function within a scope. here it is not the case, as it is implemented automatically by the language when:

```solidity
ISablierV2NFTDescriptor public override nftDescriptor;
```

**Review by PaulRBerg — APPROVED** [review]
lgtm

**Pull request merged**